server/src/main/resources/db/common/V53__alter_member_profile_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE member_profile
+ADD COLUMN birthDate date;
```

Comment:
The name of this file will need to be changed, there was a V53 in the create settings merge